string waypointname
float64 x
float64 y
float64 a
---
string result
